src/types/GlobalType.ts

```diff
@@ -21,6 +21,12 @@ export interface Image {
   image: string;
 }
 
+export enum ImportStatus {
+  UNKNOWN = 'U',
+  IMPORTED = 'I',
+  ERROR = 'E',
+}
```

Comment:
If you remove UNKNOWN (initial upload) and IMPORTED (record is deleted), you don't need this enum anymore.
You will only have records which have no errors or ones with errors so only two cases in the end :-)
As you want.

Reply:
Done